plane/source/docs/reverse-thrust-setup.rst

```diff
@@ -0,0 +1,71 @@
+
+The :ref:`THR_MAX<THR_MAX>` and :ref:`THR_MIN<THR_MIN>` values constrain the output values in non-Manual modes. A non-zero, positive value for :ref:`THR_MIN<THR_MIN>` might be used to keep an internal combustion engine running at low stick, but allow its cutoff in MANUAL mode.Note that these parameters are percentages of the output range.
+
+In order to obtain a reverse thrust using Pilot controlled reverse thrust via the throttle stick, the servo output will be setup for a reversing ESC with the maximum, trim, and minimum output values of SERVO3 setup as shown below. :ref:`THR_MIN<THR_MIN>` must be set to a negative value and will constrain the maximum reverse thrust at low stick in modes other than MANUAL to a percentage of the reverse output range. Likewise, :ref:`THR_MAX<THR_MAX>` will set the maximum positive thrust percentage in modes other than MANUAL.
+
+The :ref:`RC3_TRIM<RC3_TRIM>` and its deadzone, :ref:`RC3_DZ<RC3_DZ>`, will determine the stick position range where no thrust is produced , ie :ref:`SERVO3_TRIM<SERVO3_TRIM>`.
+
+.. figure:: ../../../images/thr-min-negative-noswitch.jpg
+    :width: 450px
+
+    Throttle Stick controls forward and reverse thrust
+
+This can be a bit awkward to use, since placement of the throttle stick in the trim deadzone to stop the motor producing thrust can be difficult. Therefore, using an RC switch option to use reverse thrust control can make this much easier, as shown below. In this case the RC channel's ``RCxOPTION`` has been set to 64. When high, this changes the throttle stick to control reverse thrust instead of forward thrust. Note, that the low stick position (and its deadzone above it) produce zero thrust in both cases and that :ref:`RC3_TRIM<RC3_TRIM>`, again is irrelevant.
+
+.. figure:: ../../../images/thr-min-negative.jpg
+    :width: 450px
+
+    Using the RC Option for changing throttle stick to control either forward or reverse thrust
+
+Reverse Thrust: AutoPilot Controlled
+------------------------------------
+
+Reverse thrust can also be used by the autopilot to steepen descents without overspeeding automatically, in automatic throttle controlled modes, including during automatic landings, by setting the :ref:`USE_REV_THRUST<USE_REV_THRUST>` parameter, as shown in the above diagrams.
+
+In order to actually use this,an airspeed sensor must be in use, and the :ref:`TECS_SINK_MAX<TECS_SINK_MAX>` must be set such that :ref:`TECS_PITCH_MIN<TECS_PITCH_MIN>` and :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` would be exceeded otherwise. The autopilot will use reverse thrust up to :ref:`THR_MIN<THR_MIN>` in order to obtain that maximum sink rate when required by the autopilot.
```

Comment:
```suggestion
In order to actually use this, an airspeed sensor must be in use, and the :ref:`TECS_SINK_MAX<TECS_SINK_MAX>` must be set such that :ref:`TECS_PITCH_MIN<TECS_PITCH_MIN>` and :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` would be exceeded otherwise. The autopilot will use reverse thrust up to :ref:`THR_MIN<THR_MIN>` in order to obtain that maximum sink rate when required by the autopilot. Typical configurations are to have :ref:`TECS_APPR_SMAX<TECS_APPR_SMAX>` higher than :ref:`TECS_SINK_MAX<TECS_SINK_MAX>` for smoother cruising while allowing steeper and more aggressive slopes for landing.
```